crates/nu-command/tests/commands/parse.rs

```diff
@@ -35,7 +35,7 @@ mod simple {
 
     #[test]
     fn double_open_curly_evaluates_to_a_single_curly() {
-        Playground::setup("parse_test_regex_2", |dirs, _sandbox| {
+        Playground::setup("parse_test_simple_2", |dirs, _sandbox| {
```

Comment:
Nit regarding the existing codebase. The following tests don't need to spin up a mock-file sandbox.

Reply:
fixed in 05d2f8429